fuse_core/include/fuse_core/autodiff_local_parameterization.h

```diff
@@ -217,7 +217,8 @@ bool AutoDiffLocalParameterization<PlusFunctor, MinusFunctor, kGlobalSize, kLoca
   return ceres::internal::AutoDiff<MinusFunctor, double, kGlobalSize, kGlobalSize>
     ::Differentiate(*minus_functor_, parameter_ptrs, kLocalSize, delta, jacobian_ptrs);
 #else
-  return ceres::internal::AutoDifferentiate<ceres::internal::StaticParameterDims<kGlobalSize, kGlobalSize>>(
+  using StaticParameters = ceres::internal::StaticParameterDims<kGlobalSize, kGlobalSize>;
+  return ceres::internal::AutoDifferentiate<kLocalSize, StaticParameters>(
```

Comment:
Yes, the new template argument is the number of residuals, which is already and still passed as an argument. So you only need to use the same value as the 3rd argument here. :+1:

Reply:
Based on the previous comment, this should probably be:
```suggestion
  using StaticParameters = ceres::internal::StaticParameterDims<kGlobalSize, kGlobalSize>;
#if !CERES_VERSION_AT_LEAST(2, 1, 0)
  return ceres::internal::AutoDifferentiate<StaticParameters>(
#else
  return ceres::internal::AutoDifferentiate<kLocalSize, StaticParameters>(
#endif
```

Reply:
As discussed offline, if we click on the `...` button in https://github.com/ceres-solver/ceres-solver/commit/e7a30359ee754057f9bd7b349c98c291138d91f4, we can see the change is in: [2.1.0](https://github.com/ceres-solver/ceres-solver/releases/tag/2.1.0)  [2.1.0rc2](https://github.com/ceres-solver/ceres-solver/releases/tag/2.1.0rc2) [2.1.0rc1](https://github.com/ceres-solver/ceres-solver/releases/tag/2.1.0rc1) [2.0.0](https://github.com/ceres-solver/ceres-solver/releases/tag/2.0.0) [2.0.0rc2](https://github.com/ceres-solver/ceres-solver/releases/tag/2.0.0rc2) [2.0.0rc1](https://github.com/ceres-solver/ceres-solver/releases/tag/2.0.0rc1)

It's also been confirmed it builds in 2.0.0.